runtime/bcverify/vrfyhelp.c

```diff
@@ -931,8 +931,16 @@ isFieldAccessCompatible(J9BytecodeVerificationData *verifyData, J9ROMFieldRef *f
 				 * the field is accessible only when it exists in the subclass or the superclass's
 				 * initialization is completed in which case uninitializedThis is set to FALSE.
 				 */
-				J9BranchTargetStack *liveStack = (J9BranchTargetStack *)verifyData->liveStack;
-				return findFieldFromCurrentRomClass(romClass, fieldRef) || !liveStack->uninitializedThis;
+				J9ROMFieldShape *field = findFieldFromCurrentRomClass(romClass, fieldRef);
```

Comment:
The new code to check `J9AccStrict` should be moved into an else relative to `if (J9_ARE_ALL_BITS_SET(receiver, BCV_SPECIAL_INIT)) {` , i.e.
```
if (J9_ARE_ALL_BITS_SET(receiver, BCV_SPECIAL_INIT)) {
}
#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
else {
     /*  findFieldFromCurrentRomClass and check for J9AccStrict */
}
#endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
```

Reply:
Thanks for the clarification. I updated the change.